src/components/AlphabetFilter/AlphabetFilter.mdx

```diff
@@ -47,7 +47,7 @@ component is addressing and the full list of items should be displayed.
 <Canvas of={AlphabetFilterStories.WithControls} />
 
 The `AlphabetFilter` composes a Chakra `Box` component, so you may pass
-any Chakra `Box` prop, as well as the props below.
+any Chakra `Box` props, as well as the props below.
```

Comment:
nit: this update isn't wrong but the singular `prop` _sounds_ more correct, but since it's react convention, leave it.